spec/integration/command_line_spec.rb

```diff
@@ -1,52 +1,54 @@
 require_relative 'integration_spec_helper'
 
-include GivenFilesystemSpecHelpers
-include CliTester
+module CommandLineSpec
```

Comment:
This is an RSpec test not a module! :see_no_evil:

Reply:
I am not clear about this. Please clarify, what I have done wrong.